web/frontend/src/pages/form/components/utils/countResult.ts

```diff
@@ -42,31 +42,20 @@ const countRankResult = (rankResult: number[][], rank: RankQuestion) => {
 // percentage of the total number of votes and which candidate(s) in the
 // select.Choices has the most votes
 const countSelectResult = (selectResult: number[][]) => {
-  const resultsInPercent: string[] = [];
-  const maxIndices: number[] = [];
-  let max = 0;
-
-  const results = selectResult.reduce((a, b) => {
-    return a.map((value, index) => {
-      const current = value + b[index];
-
-      if (current >= max) {
-        max = current;
-      }
-      return current;
+  let results: [string, number][] = [];
```

Comment:
```suggestion
  const results: [string, number][] = [];
```

`push` and `pop` are allowed on `const`. But this makes sure that you don't re-assign it a different array.